File: 20220915/모범답안/20220915_08.js/factorization.py
def GCD(a, b):
    if b == 0: return a
    else: return GCD(b, a%b)

def solution(A, B):

    gcd = GCD(A,B)
    gcdA = 0
    gcdB = 0

    while gcdA != 1:
        gcdA = GCD(A, gcd)
        A = A / gcdA

    while gcdB != 1:
        gcdB = GCD(B, gcd)
        B = B / gcdB

    if A == 1 and B == 1:
        return 1
    else:
        return 0

File: 20220915/모범답안/20220915_08.js/factorization.js
function GCD(a, b)
{
  if (b == 0)
  {
    return a;
  }
  else
  {
    return GCD(b, a%b);
  }
}

function solution(A, B)
{
  var gcd = GCD(A, B);
  var gcdA = 0;
  var gcdB = 0;

  while (gcdA != 1)
  {
    gcdA = GCD(A, gcd);
    A = A / gcdA
  }

  while (gcdB != 1)
  {
    gcdB = GCD(B, gcd);
    B = B / gcdB;
  }

  if (A == 1 && B == 1)
  {
    return 1;
  }
  else{
    return 0;
  }
}
